packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -0,0 +1,735 @@
+/* eslint-disable max-nested-callbacks */
+import { expect } from 'chai';
+import JWEditor from '../../core/src/JWEditor';
+
+async function nextTick(): Promise<void> {
+    return new Promise((resolve): number => setTimeout(resolve));
+}
+
+async function longNextTick(): Promise<void> {
+    return new Promise((resolve): void => {
+        setTimeout(resolve, 50);
+    });
+}
+
+/**
+ * Please note: devtools tests depend on one another. They cannot be run individually.
+ *
+ * There are three reasons for these tests not to be real unit tests.
+ * (1) To avoid having to create a new editor with devtools before every test, which
+ *      takes time ;
+ * (2) To avoid unnecessary complexity in the code given that various panels depend
+ *      on each other ;
+ * (3) To avoid false positives due to our lack of control on the time taken by
+ *      `longNextTick` which waits for the rendering of devtools: `expect` could be
+ *      called to early.
+ */
+
+describe('plugin-devtools', () => {
+    let wrapper: HTMLElement;
+    let devtools: HTMLElement;
+    let editor: JWEditor;
+
+    before(async () => {
+        wrapper = document.createElement('test-wrapper');
+        wrapper.style.display = 'block';
+        document.body.appendChild(wrapper);
+
+        const root = document.createElement('test-container');
+        root.style.display = 'block';
+        root.innerHTML = '<h1>Title</h1><p>a<b>b</b>c</p><p>def<br>gh<i>i</i></p><div>div</div>';
+        wrapper.appendChild(root);
+
+        Object.keys(localStorage).forEach(key => {
+            if (key.indexOf('OwlUI') !== -1) {
+                localStorage.removeItem(key);
+            }
+        });
```

Comment:
Well, then you don't test that the local storage works so this is not true 100% coverage.

Reply:
I wrote the tests for what is in plugin-devtools, here part of the code is in another folder, it will be another test which will not use this before